### DISCUSSION TITLE
October 2024 Survey Results - Question 13

### DISCUSSION THREAD

**Discussion by boc-the-git** [mealie-recipes/mealie#4651] [General]
Placeholder discussion for people to share their thoughts and insights about the survey results.

**Comment by ChipSkylark37** [comment]
I wasn't able to participate in the survey, but below are some of my comments / thoughts.

1. Recipe Finder / Discover

Using a list of websites (AllRecipes, Americas Test Kitchen, etc,) I wonder if it would be possible to have a Discovery / Recipe Finder section where it could pull the 'top rated / top commented / most popular' from those websites, and ask if you want to import them. You could have it filter based on the type of dish you want, and it goes and checks those websites.

2. Expose more of the environmental variables into the Web GUI vs Backend Config

Setting up things like SMTP, Timeout, and other settings I feel are a bit of a hassle to edit when using Docker. There may be a good reason for this, but I feel finding a way to have this moved into the Web GUI would make it a lot easier to edit.

3. Native App

Other than resources / people, is there a reason or decision on not coming out with a native iOS App? Then you could do location based reminders, notifications, and more. 

=====================================================

Echoing a lot of what others have said in the results of the survey - too many to list but Integrations is always a big one.
